README.md

```diff
@@ -47,6 +47,7 @@ Each rule has emojis denoting:
 | :two: | [no-global-expect](./docs/rules/no-global-expect.md) | disallow global expect|
 | :two: | [no-global-module-test](./docs/rules/no-global-module-test.md) | disallow global module/test/asyncTest|
 | :two: | [no-global-stop-start](./docs/rules/no-global-stop-start.md) | disallow global stop/start|
+| :two: | [no-hooks-from-ancestor-modules](./docs/rules/no-hooks-from-ancestor-modules.md) | disallow the use of hooks from ancestor modules|
```

Comment:
As far as configs go, I think this makes sense to have in recommended and two (eventually-- see next paragraph).

We can't make changes to the exported configs until a major release per the [semver policy](https://github.com/platinumazure/eslint-plugin-qunit/blob/master/README.md#semantic-versioning-policy). If you're okay with waiting until a major release for this rule to go in, that's fine, but I don't know when that will be.

I recommend not including this rule in a config in this PR; we can release this as a minor release, and then there should be an issue for v6 planning and we could make sure this gets added into recommended and two configs in the major release. Let me know what you think.